helm/alertmanager/values.yaml

```diff
@@ -17,6 +17,10 @@ config:
     receiver: webhook
     repeat_interval: 12h
 
+## Alertmanager template files to include
+#
+templateFiles: {}
+
```

Comment:
Could you provide an example keeping the content commented?

Reply:
I use something like this, shamelessly stolen from [1]. I'll add a more simpler example to the code.
```
  templateFiles:
    template_1.tmpl: |-
      {{ define "cluster" }}{{ .ExternalURL | reReplaceAll ".*alertmanager\\.(.*)" "$1" }}{{ end }}

      {{ define "slack.myorg.text" }}
      {{- $root := . -}}
      {{ range .Alerts }}
        *Alert:* {{ .Annotations.summary }} - `{{ .Labels.severity }}`
        *Cluster:*  {{ template "cluster" $root }}
        *Description:* {{ .Annotations.description }}
        *Graph:* <{{ .GeneratorURL }}|:chart_with_upwards_trend:>
        *Runbook:* <{{ .Annotations.runbook }}|:spiral_note_pad:>
        *Details:*
          {{ range .Labels.SortedPairs }} • *{{ .Name }}:* `{{ .Value }}`
          {{ end }}
      {{ end }}
      {{ end }}
```
[1] https://engineering.infinityworks.com/slack-prometheus-alertmanager/

Reply:
This si what it looks like:
![template](https://user-images.githubusercontent.com/202178/32779761-de61365a-c936-11e7-99f2-417d4af71800.jpg)